### ISSUE TITLE
code highlight gone when relaunch notepad++ 

### ISSUE THREAD

**Issue by sudhir600** [notepad-plus-plus/notepad-plus-plus#5647]
Here is the steps to reproduce this issue -

1. open any vue file, do something save but don't close
2. Now close notepadd++ application, keeping file open in tab
3. now relaunch notepad++, you will see syntax highlighter has gone.

**Before**

![before](https://user-images.githubusercontent.com/4952560/57596048-33f02180-7566-11e9-9e49-5ffcdddb2aaf.jpg)

**After**

![image](https://user-images.githubusercontent.com/4952560/57596109-85001580-7566-11e9-8235-18d6b60ed16f.png)

thanks

**Comment by MetaChuh** [comment]
@sudhir600 

.vue files are not natively supported in notepad++.
when using a .vue user defined language, notepad++ 7.6.6 restores the previous syntax highlighting of a .vue file within that session.

unfortunately you did not provide your required debug information, which vue udl you are using, and not even provide your notepad++ version number, so there is nothing we currently can and will do at the issue tracker about your experience.

please visit our notepad++ community forum to eventually work out your non source code related experience.
(link at the bottom)

thank you for your help and best regards.

-----
```
general notes:

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).
please file an issue as easy to understand and reproducible as possible.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by sudhir600** [comment]
Its not only for .vue file.. its for all file extension. I try .html file and issue was the same.

Here is the debug info-

Notepad++ v7.6.6   (64-bit)
Build time : Apr  3 2019 - 23:52:32
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : none

**Comment by MetaChuh** [comment]
@sudhir600 

**please test the following, to verify, if a clean notepad++ would work on your system.**

* download the notepad++ 7.6.6. 64 bit portable version from [>>> here <<<](https://notepad-plus-plus.org/repository/7.x/7.6.6/npp.7.6.6.bin.x64.zip).
(the portable version does not require any installation and runs completely independent and isolated from your installed version. all custom settings, as well as any stuff you might want to try at the portable version, will not modify your installed version)

* extract `npp.7.6.6.bin.x64.zip` to your desktop.

* **important note:** make sure to close all instances of notepad++ that might be running, before starting the portable version at the next step, to be sure you are using the portable version for this test.

* open the extracted `npp.7.6.6.bin.x64` folder and start this portable version by double-clicking on `notepad++.exe` inside this folder.
(note: notepad++.exe will just be seen as notepad++, if you have enabled to hide all known file extensions at your explorer settings)

* load a few files, with extensions supported by the built in highlighters (.html, .php, .json) within this portable notepad++.

* close and reopen this portable notepad++ to see, if the highlighting will be restored on a clean install of notepad++.

**Comment by sudhir600** [comment]
@MetaChuh  
This portable version is working fine. I don't know  my existing one is not working. any work around?

**Comment by MetaChuh** [comment]
@sudhir600 

glad to hear, that the clean portable version works as expected on your system.

> any work around?

yes.
the easiest way to achieve the same correct functionality, as seen at the clean portable version, is to perform an uninstall and clean install of your notepad++ 7.6.6 x64. 

* make sure you have saved all documents, closed notepad++, and you don't need any specific settings or customisations you have made at your currently installed notepad++.
* uninstall your existing notepad++ from the windows `control panel > programs and features` or the windows `settings > apps > apps & features` list.

* open `%ProgramFiles%` in windows explorer, and delete the folder `Notepad++`, if it still exists.
* open `%ProgramFiles(x86)%` in windows explorer, and delete the folder  `Notepad++`, if it exists from an older 32 bit version.
* open `%AppData%` in windows explorer, and delete the folder `Notepad++`, if it still exists.
* open `%LocalAppData%` in windows explorer, and delete the folder `Don_HO_don.h@free.fr`, if it exists.
* open `%ProgramData%` in windows explorer, and delete the folder `Notepad++`, if it exists.

* download the npp.7.6.6.Installer.x64.exe from [>>> here <<<](https://notepad-plus-plus.org/repository/7.x/7.6.6/npp.7.6.6.Installer.x64.exe) and install it.
make sure you select all your desired options when installing.

best regards.

**Comment by sudhir600** [comment]
Thanks.. fixed now
